lib/graphql-error-handler.js

```diff
@@ -0,0 +1,17 @@
+let graphqlErrorHandler = null;
+
+if (process.env.NEW_RELIC_LICENSE_KEY) {
+  const newrelic = require('newrelic');
+  graphqlErrorHandler = error => {
+    newrelic.noticeError(error, { graphql: true });
```

Comment:
@craigspaeth There’s a bunch of test failures that I’ll have to investigate first, but wanted to get some eyes on this production/NewRelic code otherwise.